src/GameLogic.cpp

```diff
@@ -182,7 +158,7 @@ void GameLogic::updateProjectileCollisions() {
 
 void GameLogic::removeFromActorsVector(Actor& actor) {
     for (int i = 0; i < actorsVector.size(); ++i) {
-        if (actor == actorsVector[i].get()) {
+        if (actor == actorsVector[i]) {
```

Comment:
Why is this changed? Because actorsVector[i] would point to the reference wrapper and not the actual actor

Reply:
Not sure, both seem to work. What do you think?

Reply:
I would have it be actorsVector[i].get() because that points to an actor versus without the .get() points to a reference wrapper

Reply:
I think either way works because c++ automatically converts the reference_wrapper to an Actor implicitly in the background, but using .get() will make more sense, as the actor is explicitly what we want the == to compare.

Reply:
Done